gui/MainWindow.cxx

```diff
@@ -959,8 +958,7 @@ void MainWindowPrivate::setActiveCamera(int id)
     {
       if (this->UI.actionSlideshowLoop->isChecked())
       {
-        this->UI.camera->setValue(*select_frames.begin());
-        this->UI.cameraSpin->setValue(*select_frames.begin());
```

Comment:
I assume the spinbox value is already updated when we update the camera value, which is why you are removing this. Is that right?

Reply:
Right.